packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -491,30 +504,43 @@ export class ExecutionEngineHttp implements IExecutionEngine {
         params: [versionedHashesHex],
       })
       .catch((e) => {
-        if (e instanceof ErrorJsonRpcResponse && parseJsonRpcErrorCode(e.response.error.code) === "Method not found") {
-          this.lastGetBlobsErrorTime = timeNow;
-          this.logger.debug("disabling engine_getBlobsV1 api call since engine responded with method not availeble", {
+        if (
+          method === "engine_getBlobsV1" &&
+          e instanceof ErrorJsonRpcResponse &&
+          parseJsonRpcErrorCode(e.response.error.code) === "Method not found"
+        ) {
+          if (method === "engine_getBlobsV1") {
+            this.lastGetBlobsV1ErrorTime = timeNow;
+          }
+          this.logger.debug(`disabling ${method} api call since engine responded with method not available`, {
             retryTimeout: GETBLOBS_RETRY_TIMEOUT,
           });
         }
         throw e;
       });
 
-    // handle nethermind buggy response
-    // see: https://discord.com/channels/595666850260713488/1293605631785304088/1298956894274060301
-    if (
-      (response as unknown as {blobsAndProofs: EngineApiRpcReturnTypes[typeof method]}).blobsAndProofs !== undefined
-    ) {
-      response = (response as unknown as {blobsAndProofs: EngineApiRpcReturnTypes[typeof method]}).blobsAndProofs;
-    }
```

Comment:
This was fixed in Nethermind [v1.30.0](https://github.com/NethermindEth/nethermind/releases/tag/1.30.0), removed per @g11tech 's comment: https://github.com/ChainSafe/lodestar/pull/7679#discussion_r2036884447